sig-list.md

```diff
@@ -21,7 +21,7 @@ Each group's material is in its subdirectory in this project.
 
 | Name | Label [1] | Chairs [2] | Contact | Meetings |
 |------|-----------|---------|---------|----------|
-|[KubeVirt Community](kubevirt-community/charter.md) |[kubevirt-community](https://github.com/kubevirt/kubevirt/labels/kubevirt-community)|<ol></ol> |[Slack](virtualization)<br/> [Mailing List](https://groups.google.com/forum/#!forum/kubevirt-dev) |<ul><li>Regular KubeVirt Community Meeting: [ Wednesday at 15:00 UTC (weekly) ](https://zoom.us/j/92221936273)</li></ul> |
+|[KubeVirt Community](kubevirt-community/charter.md) |[](https://github.com/kubevirt/kubevirt/labels/)|<ol></ol> |[Slack](virtualization)<br/> [Mailing List](https://groups.google.com/forum/#!forum/kubevirt-dev) |<ul><li>Regular KubeVirt Community Meeting: [ Wednesday at 15:00 UTC (weekly) ](https://zoom.us/j/92221936273)</li></ul> |
```

Comment:
Why? That makes the link invisible

Reply:
I think it's right to do this since the label kubevirt-community doesn't exist: https://github.com/kubevirt/project-infra/blob/main/docs/labels.md
We can argue about the flaw in the template leading to these empty edgy braces, but I think that's just a minor issue, which we can follow up on. WDYT?

Reply:
I'd prefer if the generator could put nothing there instead of a broken and unclickable link, but if it's too much work then whatever.

Reply:
I'm not saying it's too much work - I have PR https://github.com/kubevirt/community/pull/346 where I am working on (which you already reviewed) and where I'll add the change.